packages/oui-modal/README.md

```diff
@@ -8,16 +8,16 @@
 <div ng-init="$ctrl.cancel = false; $ctrl.confirm = false"
     class="oui-doc-preview-only-keep-children">
 <oui-modal
-    data-title="Modal title"
+    title="Modal title"
```

Comment:
`data-` is needed here! `title` generates a native tooltip...

Reply:
done